### ISSUE TITLE
Display a summary of total criteria being assessed

### ISSUE THREAD

**Issue by orviz** [EOSC-synergy/sqaaas-web#155]
The summary block shall be completed (along with #154 and #153) with the summary of the criteria report that follows. Here we are interested in showing the number of subcriteria fulfilled out of the total for each criterion.

A progress bar or a donut chart are good solutions to use:
#### progress bar (taken from https://clomonitor.io/)
![image](https://user-images.githubusercontent.com/1225231/160091326-6e3b7cd0-cf23-42be-a555-bd40597ae915.png)

#### donut chart (taken from GitHub)
![image](https://user-images.githubusercontent.com/1225231/160091242-0f7fd455-5431-4adb-be66-e4f0228ecfbe.png)

**Comment by orviz** [comment]
A first approach is available in the [API spec release/2.5.0](https://github.com/EOSC-synergy/sqaaas-api-spec/tree/release/2.5.0). It adds the `report:<criterion_id>:coverage` property that is an object containing:

```json
{
  "report": {
    "QC.Sty": {
      "valid": false,
      "filtered_reason": [],
      "subcriteria": {},
      "coverage": {
        "percentage": 60,
        "total_subcriteria": 5,
        "success_subcriteria": 0
      }
    }
  },
}
```

**Comment by orviz** [comment]
Released under [1.5.0](https://github.com/EOSC-synergy/sqaaas-web/releases/tag/1.5.0)
